### DISCUSSION TITLE
UI: move workspace next to settings and playground

### DISCUSSION THREAD

**Discussion by thiswillbeyourgithub** [open-webui/open-webui#4422] [General]
Hi.

Can you consider moving the workspace button next to the playground button? To me the workspace should not be much more accessible than the settings or playground and I'd rather see my chat list.

**Comment by rztaylor** [comment]
Agreed, the workspace should be accessed via the user menu to provide more space for the chat list.

**Comment by thiswillbeyourgithub** [comment]
To add: I recently created a non admin account and was surprised to see that workspace is not accessible to them. This explains a lot regarding the ability for admins to impose system prompts but nonetheless to me it's misleading to put it anywhere but in the admin settings section

**Comment by matatonic** [comment]
I have a similar proposal, remove the workspace menu entirely, and put the sub items under the 'New Chat' item in the left bar.

So the left menu bar would have:
New Chat / Chats / Prompts / Documents / Models / Playground /, etc whatever is available to the user.

I'm making some dynamic GUI coding assumptions here like having collapsed items and expanding lists under each item.

So under Chats would still be the list of recent chats, and maybe a link to a Chats page for search, import/export, select, delete, etc. but if you select Models (if available to the user), it could collapse the chats list and list models instead (or just load the models page).

**Comment by flefevre** [comment]
Given your ideas, is it compatible with a vision where we have multiple workspaces?
Thanks for your comments 

https://github.com/open-webui/open-webui/discussions/7082
